build-aci

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 tag" >/dev/stderr
+    exit 1
+fi
+
+tag=$1
+
+tgt=$(mktemp -d)
+
+# Build fleet inside
+docker run -v $PWD:/opt/fleet google/golang:1.3.1 /bin/bash -c "cd /opt/fleet && ./build"
+
+# Generate manifest into target tmp dir
+cat <<DF >${tgt}/manifest
+{
+   "acVersion" : "0.5.1",
```

Comment:
I ran the functional tests with a statically compiled fleetd binary and they pass. I tried to run the tests with the rktzd fleet but I'm having a lot of problems.

I passed rkt and stage1 to the systemd-nspawn machines, and tried to run that but I can't make it work, it's difficult to debug because there's quite a bit of Inception `coreos-vagrant -> systemd-nspawn -> rkt (another systemd-nspawn)` and I've already spent most of the day on it.

I don't know if I should spend more time on this...